karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
Improve the global structure of the project
move karma.conf.js in test folder
create a "unit" subfolder and move your unit tests in it